recipes/cotila/all/test_package/conanfile.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class CotilaTestConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
+
+    def build(self):
+        cmake = CMake(self)
+        # Current dir is "test_package/build/<build_id>" and CMakeLists.txt is
+        # in "test_package"
+        cmake.configure()
+        cmake.build()
+
+    # there are no test run, as the cotila is compile time
+    def test(self):
+        pass
```

Comment:
While cotila is compile time, the cmake script in the test package will create an executable.
```suggestion
    def test(self):
        if not tools.cross_building(self):
            bin_path = os.path.join("bin", "test_package")
            self.run(bin_path, run_environment=True)
```